src/panels/lovelace/ha-panel-lovelace.ts

```diff
@@ -66,7 +74,11 @@ class LovelacePanel extends LitElement {
     if (state === "error") {
       return html`
         <hass-error-screen title="Lovelace" .error="${this._errorMsg}">
-          <mwc-button on-click="_forceFetchConfig">Reload Lovelace</mwc-button>
+          <mwc-button on-click="_forceFetchConfig"
+            >${this.hass!.localize(
```

Comment:
Prettier won't add a new line without help, so if you add an enter after `>`, the template looks nicer.